analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -27,39 +27,43 @@ results_path="results/"
 
 
 # fusion files before and after standardization
-arriba_file="${data_path}/pbta-fusion-arriba.tsv.gz"
-starfusion_file="${data_path}/pbta-fusion-starfusion.tsv.gz"
+arriba_file="${data_path}/fusion-arriba.tsv.gz"
+starfusion_file="${data_path}/fusion-starfusion.tsv.gz"
 standard_arriba_file="${scratch_path}/arriba.tsv"
 standard_starfusion_file="${scratch_path}/starfusion.tsv"
 
 # general filtering parameters
-artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap"
+artifact_filter="GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap"
 reading_frame_filter="in-frame|frameshift|other"
 spanningFragCountFilter=100
 
 
 # relevant gene expression files
-polya_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.polya.rds"
-stranded_expression_file="${data_path}/pbta-gene-expression-rsem-fpkm.stranded.rds"
+rna_expression_file="${data_path}/gene-expression-rsem-tpm-collapsed.rds"
 
 # reference expression file
-normal_expression_file="${references_path}/Brain_FPKM_hg38_matrix.txt.zip"
+normal_expression_file="${references_path}/gtex_adrenal_gland_TPM_hg38.rds"
 
 # metadata files
-if [[ RUN_FOR_SUBTYPING == "0" ]]
+if [[ RUN_FOR_SUBTYPING -eq "0" ]]
 then
-   histologies_file="${data_path}/pbta-histologies.tsv" 
-   independent_samples_file="${data_path}/independent-specimens.wgswxs.primary-plus.tsv"
+   histologies_file="${data_path}/histologies.tsv" 
+   independent_samples_file="${data_path}/independent-specimens.wgswxspanel.primary-plus.tsv"
 else 
    histologies_file="${data_path}/pbta-histologies-base.tsv"  
-   independent_samples_file="../independent-samples/results/independent-specimens.wgswxs.primary-plus.tsv" 
+   independent_samples_file="../independent-samples/results/independent-specimens.wgswxspanel.primary-plus.tsv" 
 fi
 
 
 # data release files to use for recurrent fusion/fused genes detection
 
-putative_oncogenic_fusion="${results_path}/pbta-fusion-putative-oncogenic.tsv"
+putative_oncogenic_fusion="${results_path}/fusion-putative-oncogenic.tsv"
 
+# Run filtering code to get the reference file
+Rscript 00-normal-matrix-generation.R  --expressionMatrix $rna_expression_file \
+                                       --clinicalFile $histologies_file \
+                                       --specimenType "Adrenal Gland"
+                                       --outputfile "${references_path}/gtex_adrenal_gland_TPM_hg38.rds"
```

Comment:
```suggestion
# Run filtering code to get the reference file
Rscript 00-normal-matrix-generation.R  --expressionMatrix $rna_expression_file \
                                       --clinicalFile $histologies_file \
                                       --specimenType "Adrenal Gland" \
                                       --outputfile "${references_path}/gtex_adrenal_gland_TPM_hg38.rds"
```

Reply:
I will also modify this for all the following PRs. The README is included in PR7 but currently it hasn't mentioned this particular script - I will modify that. Thanks!

Reply:
Could you add the 00 script description here? 

Basically, by checking off the checkbox below we want to check ourselves and also let reviewer know that the chnages in each PR having required README updated 

- [ ] This analysis module has a README and it is up to date.